exercises/pangram/canonical-data.json

```diff
@@ -12,57 +12,81 @@
       ],
       "cases": [
         {
-          "description": "sentence empty",
+          "description": "can handle an empty sentence",
           "property": "isPangram",
           "input": "",
           "expected": false
         },
         {
-          "description": "pangram with only lower case",
+          "description": "handles an undefined message as empty message",
           "property": "isPangram",
-          "input": "the quick brown fox jumps over the lazy dog",
+          "input": "null",
+          "expected": false
+        },
+        {
+          "description": "recognizes a lower case pangram",
+          "property": "isPangram",
+          "input": "thequickbrownfoxjumpsoverthelazydog",
```

Comment:
Why not just test against the complete alphabet?

Reply:
Good point, never occurred to me. ^^
The input was already there so I reused it.